client/app/private-database/order/sql-database-order.html

```diff
@@ -100,52 +100,60 @@ <h2 class="mt-5" data-translate="privateDatabase_order_sql_customize_title"></h2
                         </div>
                         <div class="alert alert-warning" role="alert"
                              data-translate="privateDatabase_order_step1_hosting_warning"
-                             data-ng-if="sqlOrderCtrl.isPrivateDb() && sqlOrderCtrl.model.datacenter">
+                             data-ng-if="$ctrl.isPrivateDb() && $ctrl.model.datacenter">
                         </div>
                     </div>
 
                     <!-- DataCenter -->
-                    <div data-ng-if="(sqlOrderCtrl.isPrivateDb() && sqlOrderCtrl.model.hosting === sqlOrderCtrl.noHostValue) || sqlOrderCtrl.isCloudDb()">
+                    <div data-ng-if="($ctrl.isPrivateDb() && $ctrl.model.hosting === $ctrl.noHostValue) || $ctrl.isCloudDb()">
                         <label class="required" for="orderDc"
                                data-translate="privateDatabase_order_step1_datacenter"></label>
                         <div class="oui-select oui-select_inline">
                             <select class="oui-select__input" id="orderDc" name="orderDc" required
-                                    data-ng-change="sqlOrderCtrl.getDuration()"
-                                    data-ng-model="sqlOrderCtrl.model.datacenter"
-                                    data-ng-options="dc as ('privateDatabase_dashboard_datacenter_' + dc | translate) for dc in sqlOrderCtrl.getData(sqlOrderCtrl.model.type).datacenters track by dc">
+                                    data-ng-change="$ctrl.getDuration()"
+                                    data-ng-model="$ctrl.model.datacenter"
+                                    data-ng-options="dc as ('privateDatabase_dashboard_datacenter_' + dc | translate) for dc in $ctrl.getData($ctrl.model.type).datacenters track by dc">
                                 <option value="" disabled
                                         data-translate="select_option"></option>
                             </select>
                             <span class="oui-icon oui-icon-chevron-down" aria-hidden="true"></span>
                         </div>
                         <div class="alert alert-warning" role="alert"
                              data-translate="privateDatabase_order_step1_no_hosting_warning"
-                             data-ng-if="sqlOrderCtrl.isPrivateDb() && sqlOrderCtrl.model.hosting === sqlOrderCtrl.noHostValue">
+                             data-ng-if="$ctrl.isPrivateDb() && $ctrl.model.hosting === $ctrl.noHostValue">
                         </div>
                     </div>
 
-                    <div class="loader mt-5 text-center" data-ng-if="sqlOrderCtrl.loading.durations"></div>
+                    <div class="alert alert-warning" role="alert"
+                         data-ng-if="$ctrl.isPrivateDb() && $ctrl.selectedHosting && $ctrl.selectedHosting.stillHasFreeDbOffer">
+                        <h3 class="h5" data-translate="privateDatabase_order_sql_included_title"></h3>
```

Comment:
https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/ovh-ui-kit/typography

Reply:
`oui-header_5` is available.

https://github.com/ovh-ux/ovh-ui-kit/pull/296/files#diff-bc7d9dc2b0a922de7ac1252784ab8615R37